integration/walletextension/wallet_extension_test.go

```diff
@@ -36,10 +38,8 @@ const (
 	dummyAccountAddr = "0x8D97689C9818892B700e27F316cc3E41e17fBeb9"
 
-	startPort              = 3000
-	httpProtocol           = "http://"
-	pathGenerateViewingKey = "/generateviewingkey/"
```

Comment:
Some more clean-up. I used the existing constants defined in `tools/walletextension`, rather than redefining them here.